tools/app_interface_reporter.py

```diff
@@ -34,9 +35,13 @@
 CONTENT_FORMAT_VERSION = '1.0.0'
 DASHDOTDB_SECRET = os.environ.get('DASHDOTDB_SECRET',
                                   'app-sre/dashdot/auth-proxy-production')
+APP_SRE_PROD_01_PROMETHEUS_URL = \
+    os.environ.get('APP_SRE_PROD_01_PROMETHEUS_URL',
+                   'https://prometheus.app-sre-prod-01.devshift.net')
+TEKTON_JOBS_CLUSTER = os.environ.get('TEKTON_JOBS_CLUSTER', 'app-sre-prod-01')
```

Comment:
Likewise here I believe you can query pipelines-providers with `provider: tekton` to get this and then get to the cluster schema which will have a `prometheusUrl` field